### ISSUE TITLE
greenkeeper badge removal

### ISSUE THREAD

**Issue by travi** [gr2m/migrate-greenkeeper-to-dependabot-cli#1]
looks like you already have it solved, but if you'd want to tinker with remark, i built [a plugin](https://github.com/form8ion/remark-remove-greenkeeper-badge) for removing the greenkeeper badge. My usage is [here](https://github.com/form8ion/remove-greenkeeper/blob/master/src/badge.js)

**Comment by gr2m** [comment]
very interesting, I didn't know about remark plugins!

**Comment by travi** [comment]
yeah, its been fun to play with ASTs in a bit different context. I also created one [for injecting new badges](https://github.com/form8ion/remark-inject-badges), but its a bit more specific to my README conventions.
